.bldr.toml

```diff
@@ -176,6 +176,7 @@ paths = [
   "components/common/*",
   "components/hab/*",
   "components/pkg-export-docker/*"
```

Comment:
Is a comma not needed here?

Reply:
In case people wanna install a syntax checker, [this](https://github.com/vmchale/tomlcheck) seems to be the only one.

Reply:
Indeed it is. Thanks for catching this. I don't have any means to test my changes to the CI (or i don't know them).

Reply:
but yeah, I can at least test the toml syntax with that tool. :)

Reply:
That tool sucks btw. if you make a mistake in toml, it doesn't tell you where exactly but still useful to verify that you've fixed everything, once you figure out on your own what's wrong. :)